README.md

```diff
@@ -1,6 +1,227 @@
+
+## Docs
+
+[Documentation](/app-services-sdk-js/docs) 
+
+## Examples
+
+[Examples](/app-services-sdk-js/examples) 
+
+# RHOAS SDK for Python
+
+Python packages and API clients for Red Had OpenShift Application Services (RHOAS) 
+
+## Prequisites
+
+- [Python 3.9](https://docs.python.org/3/) or above
+- [pip](https://pypi.org/project/pip/) for installing packages
+
+## Installation
+
+Currently all RHOAS SDKs are bundled together. To install the RHOAS SDK with the pip package installer:
+
+```shell
+$ python -m pip install rhoas-sdks
+```
+
+## RHOAS App Services SDK for Python
+
+> NOTE: Some of these APIs are under development and may sometimes cause backwards-incompatible changes.
+
+All packages are now available and can be accessed by just importing them as shown below:
+
+
+| API                       | Status | Package                                                                                                                                                         |
+| :------------------------ | ------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [KafkaManagement](/app-services-sdk-python/sdks/kafka_mgmt_sdk/README.md)           | beta   | `import python rhoas_kafka_mgmt_sdk`          |
+| [ServiceRegistryManagement](app-services-sdk-python/sdks/registry_mgmt_sdk/README.md)  | alpha   | `import rhoas_service_registry_mgmt_sdk`         |
+| [ConnectorManagement](app-services-sdk-python/sdks/connector_mgmt_sdk/README.md)       | alpha  | `import rhoas_connector_mgmt_sdk`  |
+| [ServiceAccounts](app-services-sdk-python/sdks/service_accounts_mgmt_sdk/README.md) | alpha | `import rhoas_service_accounts_mgmt_sdk` |
+
+ 
+ ## Instances SDKs
+
+| API              | Status | Package                                                                                                                                                                               |
+| ---------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [KafkaInstance](app-services-sdk-python/sdks/kafka_instance_sdk/README.md)    | beta   | `import rhoas_kafka_instance_sdk`|
+| [RegistryInstance](app-services-sdk-python/sdks/registry_instance_sdk/README.md) | beta   | `import rhoas_registry_instance_sdk` |
+
+
+## Documentation
+[Documentation](/app-services-sdk-python/docs)
+## Examples
+[Examples](/app-services-sdk-python/examples)
+
+# RHOAS SDK for Java
+
+Java packages and API clients for RHOAS services
+
+## Prequisites
+
+- [NodeJS 14.x lts](https://nodejs.org/en/about/releases/) or above
+- [Maven 3.6.x](https://maven.apache.org) or above
+- [YQ 4.6.x](https://github.com/mikefarah/yq#install) or above
```

Comment:
Why?

Reply:
These are requirements for developing I will move these to `CONTRIBUTING`